Makefile

```diff
@@ -78,7 +82,7 @@ lint-ruby:  # lints the Ruby files
 
 setup:  # the setup steps necessary on developer machines
 	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-	go get -u github.com/onsi/ginkgo/ginkgo
+	go get -u github.com/onsi/ginkgo/ginkgo github.com/DATA-DOG/godog/cmd/godog
```

Comment:
Do we still need to install ginkgo here? I don't think so since we aren't using the cli

Reply:
We are still running the Ginkgo test framework that prints the colorful test output. I think that's in the ginkgo package as well. It seems safer to stop installing this once we eliminate all Ginkgo test suites.

Reply:
Ginkgo already exists in Gopkg.lock so we are installing already anyway. Believe this was here only for being able to use the binary. Fine as is since we plan to remove gingko fully soon